docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -89,11 +69,19 @@ Use API keys to authenticate with:
 - [The Cloud Operations API](/production-deployment/cloud/operation-api.mdx)
 - [Temporalʼs Terraform provider](/production-deployment/cloud/terraform-provider)
 
+### API key permissions
+
+API keys support both users and Service Accounts.
+Here are the differences in their permissions:
+
+- Any user can create, delete, and update their own API key access using the Cloud UI or `tcld`.
+- Only Global Administrators and Account Owners can create, delete, and update access to API keys for Service Accounts.
```

Comment:
we'll have to remember to update this when we allow namespace admins to create API keys / service accounts :P